.github/workflows/my_plugin.yaml

```diff
@@ -21,11 +21,12 @@ jobs:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/flutter_package.yml@v1
     with:
       flutter_channel: stable
-      flutter_version: 3.3.2
+      flutter_version: 3.7.1
       working_directory: src/my_plugin/my_plugin
 
   android:
     runs-on: macos-latest
+    if: false # TODO: Integration tests are flaky since Flutter 3.7.0
```

Comment:
Thoughts on leaving these enabled but just removing them from the required status checks in GitHub? That way we can keep an eye on whether the issue is resolved.

Reply:
For the record: We decided to keep this line as we cant disable specific workflows on gh settings